cmd/check_container.go

```diff
@@ -99,13 +118,89 @@ var checkContainerCmd = &cobra.Command{
 			return err
 		}
 
+		// submitting results to pxysis if submit flag is set
+		if s, _ := cmd.Flags().GetBool("submit"); s {
+			log.Info("preparing results that will be submitted to Red Hat")
+
+			certImageJsonFile, err := os.Open(path.Join(artifacts.Path(), CertImageFileName))
+			defer certImageJsonFile.Close()
+			if err != nil {
+				return err
+			}
+
+			certImageBytes, err := ioutil.ReadAll(certImageJsonFile)
+			if err != nil {
+				return err
+			}
+
+			certImage := new(pyxis.CertImage)
+			err = json.Unmarshal(certImageBytes, &certImage)
+			if err != nil {
+				return err
+			}
+
+			rpmManifestJsonFile, err := os.Open(path.Join(artifacts.Path(), RPMManifestFileName))
+			defer rpmManifestJsonFile.Close()
+			if err != nil {
+				return err
+			}
+
+			rpmManifestBytes, err := ioutil.ReadAll(rpmManifestJsonFile)
+			if err != nil {
+				return err
+			}
+
+			rpmManifest := new(pyxis.RPMManifest)
+			err = json.Unmarshal(rpmManifestBytes, rpmManifest)
+			if err != nil {
+				return err
+			}
+
+			testResultsJsonFile, err := os.Open(path.Join(artifacts.Path(), TestResultsFileName))
+			defer testResultsJsonFile.Close()
+			if err != nil {
+				return err
+			}
+
+			testResultsBytes, err := ioutil.ReadAll(testResultsJsonFile)
```

Comment:
`ioutil` is deprecated in Go 1.16 (https://go.dev/doc/go1.16#ioutil). We can use `io` or `os` packages instead.

Reply:
I made this change, just an FYI they call `io.ReadAll()` directly now. See [here](https://github.com/golang/go/blob/master/src/io/ioutil/ioutil.go#L25)